packages/components/src/popover/index.tsx

```diff
@@ -406,17 +364,16 @@ const UnforwardedPopover = (
 			fallbackReferenceElement,
 			fallbackDocument: document,
 		} );
-		const scale = getFrameScale( resultingReferenceOwnerDoc );
+
 		const resultingReferenceElement = getReferenceElement( {
 			anchor,
 			anchorRef,
 			anchorRect,
 			getAnchorRect,
 			fallbackReferenceElement,
-			scale,
 		} );
 
-		referenceCallbackRef( resultingReferenceElement );
+		refs.setReference( resultingReferenceElement );
```

Comment:
This appears to be causing an error in the site editor:

![Error message](https://github.com/WordPress/gutenberg/assets/8436925/b7ed439f-fbba-4847-8d8e-51dd66a00035)

To reproduce:

* Open the site editor
* Roll over some blocks
* You should see the error.

Funny enough, this is also caught by a couple of TS errors (at least in my local environment):

![ts-error](https://github.com/WordPress/gutenberg/assets/8436925/af552ec7-c593-45ba-a96c-de764eb4648c)

Reply:
That's very strange, are you sure you ran `npm install` after checking out this branch? The `refs.setReference` API is new in `@floating-ui/react-dom@2.0.0`. Seems like you don't have it installed, and neither the new types.

Reply:
Funny enough I did run `npm install`, but now after running it again, it worked well. The error is also gone. Disregard. 👍